### ISSUE TITLE
Update Problem

### ISSUE THREAD

**Issue by jonathanwadimir** [stuartridout/teamlink#47]
Hello,
 
Please I ask you for help to update Teamlink.
 
This pops up:

![image](https://user-images.githubusercontent.com/14363481/204365745-0d134df0-8b14-4127-a2bc-3aa69657c9a6.png)

**Comment by stuartridout** [comment]
Have you completed the initial configuration?

**Comment by jonathanwadimir** [comment]
yes, it's already working in our organization. but even when we create a new user and assigned a group he can´t config the initial setup

**Comment by jonathanwadimir** [comment]
Dear, is it possible your help for the case?

**Comment by stuartridout** [comment]
Hi there.  The person may have access to the app but not the data.   Are they a member of the team who have access to the data?
